### PR TITLE
Add ruby examples for triggers on lambda (address #1740)

### PR DESCRIPTION
**Pull request by grzegorz-wcislo**
### Affected components 

- Community Content

### Related Issues

#1740 

### Steps to test and verify

Follow the READMEs

### CODE REVIEW

**Comment by grzegorz-wcislo** [comment]
Thank you for your feedback! :smile:

I think this env variable is called `HASURA_GRAPHQL_ADMIN_SECRET` on the server side. Could you try that?

If it doesn't work replace `rescue` with

```ruby
rescue StandardError => e
  puts e.message
  puts e.backTrace
```

And please paste the AWS request logs here.

I have only tested this using local server with ngrok, not heroku,

**Comment by marionschleifer** [comment]
Hey, yes, I'm using the correct env variable on the server side. There are no errors in the AWS request logs as far as I can see.

Would you mind testing this with Heroku and see if it works for you? Otherwise, you can ping me on our Discord channel (@marion#9554) and we can debug together 🙂

**Comment by grzegorz-wcislo** [comment]
Hi @marionschleifer, I believe I have found a solution.

Turns out it was a ssl issue (I was using http on localhost). It should work fine now :smile:

**Comment by marionschleifer** [comment]
@grzegorz-wcislo have you tried this with Heroku? I still get the same error 🤔

**Comment by grzegorz-wcislo** [comment]
Yes, I did. I used this guide to create the dyno: https://docs.hasura.io/1.0/graphql/manual/getting-started/heroku-simple.html.

Could you try to print the error message (I have provided a snippet above) and send me AWS request logs? :)

I don't want to clutter example code with a lot of error handling, that's why there is only one error message :O

**Comment by marionschleifer** [comment]
Sure, here they are:

image

**Comment by grzegorz-wcislo** [comment]
Thanks for your help :grin:

Now I'm pretty sure the issue is solved :smile: I have made a mistake in the docs, it should say `update trigger` instead of `insert trigger`. Could you verify that the update trigger works for you?

I will fix the readme right away :smile:

File: community/boilerplates/event-triggers/aws-lambda/ruby/mutation/lambda_function.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+require 'json'
+require 'net/http'
+
+def lambda_handler(event:, context:)
+  access_key = ENV['ACCESS_KEY']
+  hge_endpoint = ENV['HGE_ENDPOINT']
+  query = <<~QUERY
+    mutation updateNoteRevision ($noteId: Int!, $data: String!) {
+      insert_note_revisions (objects: [
```
Comment (marionschleifer) [inline community/boilerplates/event-triggers/aws-lambda/ruby/mutation/lambda_function.rb]: This should be `insert_note_revision`. It worked for me after changing it to that 🙂

**Comment by grzegorz-wcislo** [comment]
All done! :)

**Comment by marionschleifer** [comment]
I don't see any changes since my last review 🤔

**Comment by grzegorz-wcislo** [comment]
@marionschleifer Oh, I didn't notice my push was rejected. Now it's all done :smile:

**Pull request merged**